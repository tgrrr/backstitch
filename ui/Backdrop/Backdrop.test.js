import React from 'react'
import renderer from 'react-test-renderer'
import { Backdrop } from '.'

it('Backdrop: default', () => {
    const component = renderer.create(
        <Backdrop>
            <>
                Backdrop
            </>
        </Backdrop>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
