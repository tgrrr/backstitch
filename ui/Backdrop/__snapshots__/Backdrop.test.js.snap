// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Backdrop "
    data-testid="Backdrop"
  >
    <div
      aria-hidden={true}
      className="MuiBackdrop-root"
      style={
        Object {
          "opacity": 0,
          "visibility": "hidden",
        }
      }
    >
      Backdrop
    </div>
  </div>
</div>
`;
